java/src/jmri/jmrit/beantable/LRouteTableAction.java

```diff
@@ -970,7 +970,7 @@ public void actionPerformed(ActionEvent e) {
             tab2.add(new JLabel(rbx.getString("OutputTitle")));
             _outputAllButton = new JRadioButton(Bundle.getMessage("All"), true);
             JRadioButton includedOutputButton = new JRadioButton(Bundle.getMessage("Included"), false);
-            tab2.add(makeShowButtons(_outputAllButton, includedOutputButton, null, Bundle.getMessage("Show")));
+            tab2.add(makeShowButtons(_outputAllButton, includedOutputButton, null, "Show")); // NOI18N
```

Comment:
Well, it's going in 4.3.9 (and, therefore, 4.4) like this so should be reviewed for inclusion in 4.5.1.